packet/models.go

```diff
@@ -129,6 +129,8 @@ type Instance struct {
 	SSHKeys []string `json:"ssh_keys,omitempty"`
 	// CustomData
 	NetworkReady bool `json:"network_ready,omitempty"`
+	// Boot Drive hint for VMware Installer
```

Comment:
Typically a doc comment like this "[begins with the name of the item it describes](https://golang.org/doc/effective_go#commentary)".  Something like:
`// BootDriveHint defines what the VMware installer should use as the booting drive.`

Reply:
None of the doc comments in the preceding 20 lines seem to conform to that standard, but that's no reason not to start now. Will fix.